### ISSUE TITLE
`forc` should print out the error if `fuel-core` fails to start, instead of swallowing any logs.

### ISSUE THREAD

**Issue by emilyaherbert** [FuelLabs/sway#742]
This should either be fixed or there should be a better error telling the user how to set it up themselves.

```rust
$ forc run
  Compiled library "core".
  Compiled library "core".
warning
 --> FuelLabs-sway-lib-std-57205cb/src/result.sw:0:5
  |
...
3 |     Ok: T,
  |     -- Enum variant Ok is never constructed.
  |
warning
 --> FuelLabs-sway-lib-std-57205cb/src/result.sw:0:5
  |
...
4 |     Err: E,
  |     --- Enum variant Err is never constructed.
  |
  Compiled library "lib-std" with 2 warnings.
  Compiled script "my-fuel-project".
  Bytecode size is 28 bytes.
We noticed you don't have fuel-core running, would you like to start a node [y/n]?y
Error: "Could not start fuel-core"
```

**Comment by sezna** [comment]
If you run `fuel-core` in your terminal, what is the output? This looks like there was an error launching `fuel-core`

**Comment by mohammadfawaz** [comment]
I've seen this before but I think back then I didn't have `fuel-core` installed. Now that I have it installed, running `forc run` in one of the test directories works fine:
```
$ forc run
  Downloading "core-http://github.com/FuelLabs/sway-lib-core" ("/home/mohammad/.forc/7786363458449203048/v0.0.1")
  Compiled library "core".
  Downloading "std-http://github.com/FuelLabs/sway-lib-std" ("/home/mohammad/.forc/4024910796088766844/v0.0.1")
  Downloading "core-http://github.com/FuelLabs/sway-lib-core" ("/home/mohammad/.forc/7786363458449203048/default")
  Compiled library "core".
warning
 --> FuelLabs-sway-lib-std-5a0938f/src/result.sw:4:5
  |
2 | 
3 | enum Result<T, E> {
4 |     Ok: T,
  |     -- Enum variant Ok is never constructed.
5 |     Err: E,
6 | }
  |
warning
 --> FuelLabs-sway-lib-std-5a0938f/src/result.sw:5:5
  |
3 | 
4 |     Ok: T,
5 |     Err: E,
  |     --- Enum variant Err is never constructed.
6 | }
  |
  Compiled library "lib-std" with 2 warnings.
  Compiled script "multi_item_import".
  Bytecode size is 140 bytes.
We noticed you don't have fuel-core running, would you like to start a node [y/n]?y
[Return { id: 0x0000000000000000000000000000000000000000000000000000000000000000, val: 0, pc: 572, is: 464 }, ScriptResult { result: InstructionResult { reason: RESERV00, instruction: Instruction { op: 0, ra: 0, rb: 0, rc: 0, rd: 0, imm06: 0, imm12: 0, imm18: 0, imm24: 0 } }, gas_used: 0 }]
```

**Comment by sezna** [comment]
We found out that the `~/.fuel` database was out of date in Emily's case. We should keep this issue open but change its title to something about `forc` eating the error.

**Comment by mitchmindtree** [comment]
Going to close this as it seems to have been addressed by #786 (feel free to re-open if I'm wrong!)
